packages/react-components/src/Labelled.tsx

```diff
@@ -147,5 +148,46 @@ export default React.memo(styled(Labelled)`
         }
       }
     }
+
+    & > .ui--Labelled-content > .ui--Static {
+      ${(props) => props.modalNew ? 'border: 1px solid #DFDFDF;' : ''}
+      ${(props) => props.modalNew ? 'background-color: #FFFFFF;' : ''}
+      ${(props) => props.modalNew ? 'color: #8B8B8B;' : ''}
+    }
+  }
+  
+  .ui--CreateAccount-new-create-modal div.ui--Modal-Column > &.ui--Labelled,  .ui--CreateAccount-new-create-modal div:not(.ui--Modal-Column) > &.ui--Labelled {
```

Comment:
Ad per crsoo package css - this will get really mess if we have the specific CSS everywhere for each modal.